index.html

```diff
@@ -42,19 +41,42 @@
           <a href="#">Join</a>
         </li>
         <li>
-          <a href="#">Get Tickets</a>
+          <a href="#" class="tickets">Get Tickets</a>
         </li>
       </ul>
     </div>
+    <ul class="mobile-header-nav-links">
+      <li>
+        <i class="fa fa-times close"></i>
+      </li>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a href="#">Program</a>
+      </li>
+      <li>
+        <a href="#">Join</a>
+      </li>
+      <li>
+        <a href="#" class="tickets">Get Tickets</a>
+      </li>
+    </ul>
+    <div class="menu">
+      <i class="fa fa-bars"></i>
+    </div>
   </header>
-  <section class="main">
 
-      <p class="welcome-text">"Welcome Game of Thrones Fans!!"</p>
-      <h1>Game of Thrones <br> Fans Summit <br> 2021</h1>
-      <p class="explanation">A joyful celebration about the now concluded and successful Game of Thrones series, released by H.B.O. The show's first episode premiered on and the last episode aired . It has been considered as the greatest series by a lot of people.</p>
-      <h2>2021.09.12(Sun) ~ 2021.09.18(Sat)</h2>
-      <h3>@ Central park, NewYork, America and London.</h3>
+  <section class="main">
+    <div class="main-background">
 
+    </div>
+    <p class="welcome-text">"Welcome Game of Thrones Fans!!"</p>
+    <h1>Game of Thrones <br> Fans Summit <br> 2021</h1>
+    <p class="mobile-explanation">A joyful celebration about the now concluded and successful Game of Thrones series, released by H.B.O.</p>
+    <p class="explanation">A joyful celebration about the now concluded and successful Game of Thrones series, released by H.B.O. The show's first episode premiered on and the last episode aired . It has been considered as the greatest series by a lot of people.</p>
+    <h2>2021.09.12(Sun) ~ 2021.09.18(Sat)</h2>
+    <h3>@ Central park, NewYork, America and London.</h3>
   </section>
```

Comment:
- [ ] Let us improve our styling to the first section kindly use the same font, size and family they provided in the template.
-----------------------------------------Original------------------------------------------

![rev5](https://user-images.githubusercontent.com/55635977/133495467-6de147d8-d3be-4e95-a173-72b705a4126e.png)
-----------------------------------------Yours------------------------------------------

![rev6](https://user-images.githubusercontent.com/55635977/133494682-32baca4e-400c-4177-b38f-4d6111d5111a.png)

Reply:
This issue needs further improvement. 

The **alignment** and **typography**  of your _hero_ / _introduction_ section do not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve upon these:
- [ ] Add appropriate **padding at the top** of the contents of this section, as seen in the original design
- [ ] The typography(**font sizes**, **font-weight**, **font color**) of the _small line of text_, the _page title_,  _block of text_, _medium line of text in bold letters_, and the _smaller line of text_, should closely match the provided template design.
- [ ] Contents of this section should be **evenly spaced** (paddings/ margins), as can be seen in the provided template design.

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/133642813-230ed788-9ff8-4bc0-9dbb-c982196651bc.png)


**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)